index.html

```diff
@@ -11,5 +11,38 @@
     <title>My Portfolio2</title>
 </head>
 <body>
+  <div class="gen-container">
+    <nav>
+      <h3 class="logo">
+        My Logo
+      </h3>
+      <div class="menu-bars">
+        <img src="header-section\bars.svg" alt="">
+      </div>
+    </nav>
+    <div class="container"> 
+      <h1>
+        I'm Lugard<br>
+        Glad to see <br> 
+        you!
+      </h1>
```

Comment:
Kindly fix the `text-color` of this heading to match the Figma design, it should be `172B4D` 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/192273344-1fb10c8f-6f88-49c8-b444-7596d094cadc.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/192273384-a165486f-b1e7-4a83-91b0-4692ecc46994.png)